go-auth.go

```diff
@@ -374,6 +374,8 @@ func setCache(authOpts map[string]string) {
 
 	var aclCacheSeconds int64 = 30
 	var authCacheSeconds int64 = 30
+	var authJitterSeconds int64 = 3
```

Comment:
As almost always, this one's on me: `jitters` are only known to `cache`, there's no need to have them processed in the `main` package. Then again, `acl/authCacheSeconds` shouldn't be there either, but refactoring `cache` to process the option and extract the duration is totally out of scope for this PR, so leave it in `main` and we can take care of it later.

TL;DR: never mind this comment. And note to self: don't start open-source projects when you've only used the language for a couple of weeks, you'll regret it many years later when making/reviewing changes (and have a desire to burn the whole ugly, bad-designed thing down while we are at it). :woman_facepalming: